File: vignettes/fmelt.c
'x' must be an integer
'n' must be a positive integer
Argument to 'which' must be logical
concat: 'vec' must be a character vector
concat: 'idx' must be an integer vector of length >= 0
Internal error in concat: 'idx' must take values between 1 and length(vec); 1 <= idx <= %d
Unknown 'measure.vars' type %s at index %d of list
id.vars and measure.vars are internally guessed when both are 'NULL'. All non-numeric/integer/logical type columns are considered id.vars, which in this case are columns [%s]. Consider providing at least one of 'id' or 'measure' vars in future.
Unknown 'id.vars' type %s, must be character or integer vector
One or more values in 'id.vars' is invalid.
'measure.vars' is missing. Assigning all columns other than 'id.vars' columns as 'measure.vars'.\n
Assigned 'measure.vars' are [%s].\n
Unknown 'measure.vars' type %s, must be character or integer vector/list
One or more values in 'measure.vars' is invalid.
'id.vars' is missing. Assigning all columns other than 'measure.vars' columns as 'id.vars'.\n
Assigned 'id.vars' are [%s].\n
Unknown 'id.vars' type %s, must be character or integer vector
One or more values in 'id.vars' is invalid.
Unknown 'measure.vars' type %s, must be character or integer vector
One or more values in 'measure.vars' is invalid.
When 'measure.vars' is a list, 'value.name' must be a character vector of length =1 or =length(measure.vars).
When 'measure.vars' is either not specified or a character/integer vector, 'value.name' must be a character vector of length =1.
'variable.name' must be a character/integer vector of length=1.
variable_table attribute of measure.vars should be a data table with at least one column
variable_table attribute of measure.vars should be a data table with same number of rows as max length of measure.vars vectors =%d
variable_table attribute of measure.vars should be either NULL or a data table
Internal error: combineFactorLevels in fmelt.c expects all-character input
Internal error: combineFactorLevels in fmelt.c expects a character target to factorize
'measure.vars' [%s] are not all of the same type. By order of hierarchy, the molten data value column will be of type '%s'. All measure variables not of type '%s' will be coerced too. Check DETAILS in ?melt.data.table for more on coercion.\n
Unknown column type '%s' for column '%s'.
Internal error: fmelt.c:getvarcols %d %d
%d
%d
variable_table does not support column type '%s' for column '%s'.
Unknown column type '%s' for column '%s' in 'data'
Input is not of type VECSXP, expected a data.table, data.frame or list
Argument 'value.factor' should be logical TRUE/FALSE
Argument 'variable.factor' should be logical TRUE/FALSE
Argument 'na.rm' should be logical TRUE/FALSE.
Argument 'variable.name' must be a character vector
Argument 'value.name' must be a character vector
Argument 'verbose' should be logical TRUE/FALSE
ncol(data) is 0. Nothing to melt. Returning original data.table.
names(data) is NULL. Please report to data.table-help

File: vignettes/fmelt.R
The melt generic in data.table has been passed a
, but data.table::melt currently only has a method for data.tables. Please confirm your input is a data.table, with setDT(
) or as.data.table(
). If you intend to use a method from reshape2, try installing that package first, but do note that reshape2 is superseded and is no longer actively developed.
The melt generic in data.table has been passed a
and will attempt to redirect to the relevant reshape2 method; please note that reshape2 is superseded and is no longer actively developed, and this redirection is now deprecated. To continue using melt methods from reshape2 while both libraries are attached, e.g. melt.list, you can prepend the namespace like reshape2::melt(
). In the next version, this warning will become an error.
Input patterns must be of type character.
Pattern
s
not found: [
,
]
both sep and pattern arguments used in measure; must use either sep or pattern (not both)
multiple.keyword must be a character string with nchar>0
cols must be a character vector of column names
each ... argument to measure must be either a symbol without argument name, or a function with argument name, problems:
,
group names specified in ... conflict with measure argument names; please fix by changing group names:
,
names should be unique, problems:
,
number of ... arguments to measure =
must be same as
=
pattern must be character string
pattern did not match any cols, so nothing would be melted; fix by changing pattern
pattern must contain at least one capture group (parenthesized sub-pattern)
sep must be character string
each column name results in only one item after splitting using sep, which means that all columns would be melted; to fix please either specify melt on all columns directly without using measure, or use a different sep/pattern specification
number of unique column IDs =
is less than number of melted columns =
; fix by changing pattern/sep
each ... argument to measure must be a function with at least one argument, problem:
each ... argument to measure must be a function that returns an atomic vector with same length as its first argument, problem:
conversion function returned vector of all NA
number of unique groups after applying type conversion functions less than number of groups, change type conversion
column class=
after applying conversion function, but must be character
is the only group; fix by creating at least one more group
'data' must be a data.table
'value.name' provided in both 'measure.vars'
and 'value.name argument'; value provided in
'measure.vars' is given precedence.
Please provide a name to each element of 'measure.vars'.
The melt generic in data.table has been passed a
, but data.table::melt currently only has a method for data.tables. Please confirm your input is a data.table, with setDT(
) or as.data.table(
). If you intend to use a method from reshape2, try installing that package first, but do note that reshape2 is superseded and is no longer actively developed.
The melt generic in data.table has been passed a
and will attempt to redirect to the relevant reshape2 method; please note that reshape2 is superseded and is no longer actively developed, and this redirection is now deprecated. To continue using melt methods from reshape2 while both libraries are attached, e.g. melt.list, you can prepend the namespace like reshape2::melt(
). In the next version, this warning will become an error.
Input patterns must be of type character.
Pattern
s
not found: [
,
]
both sep and pattern arguments used in measure; must use either sep or pattern (not both)
multiple.keyword must be a character string with nchar>0
cols must be a character vector of column names
each ... argument to measure must be either a symbol without argument name, or a function with argument name, problems:
,
group names specified in ... conflict with measure argument names; please fix by changing group names:
,
names should be unique, problems:
,
number of ... arguments to measure =
must be same as
=
pattern must be character string
pattern did not match any cols, so nothing would be melted; fix by changing pattern
pattern must contain at least one capture group (parenthesized sub-pattern)
sep must be character string
each column name results in only one item after splitting using sep, which means that all columns would be melted; to fix please either specify melt on all columns directly without using measure, or use a different sep/pattern specification
number of unique column IDs =
is less than number of melted columns =
; fix by changing pattern/sep
each ... argument to measure must be a function with at least one argument, problem:
each ... argument to measure must be a function that returns an atomic vector with same length as its first argument, problem:
conversion function returned vector of all NA
number of unique groups after applying type conversion functions less than number of groups, change type conversion
column class=
after applying conversion function, but must be character
is the only group; fix by creating at least one more group
'data' must be a data.table
'value.name' provided in both 'measure.vars'
and 'value.name argument'; value provided in
'measure.vars' is given precedence.
Please provide a name to each element of 'measure.vars'.
